packages/conf-neko/conf-neko.1/opam

```diff
@@ -9,9 +9,9 @@ build: [["neko" "-version"]]
 depexts: [
   [["alpine"] ["neko-dev"]]
   [["debian"] ["neko-dev"]]
```

Comment:
Would you also add `neko` to debian?

Reply:
That would be my guess, but I don't have a Debian system to verify on.

Reply:
I'm certain since the ubuntu package comes from debian.

Reply:
Ok, I added a second commit that adds the Debian depext.